hack/lib/test.sh

```diff
@@ -83,7 +83,7 @@ kube::test::object_assert() {
   for j in $(seq 1 "${tries}"); do
     # shellcheck disable=SC2086
     # Disabling because "args" needs to allow for expansion here
-    res=$(eval kubectl get "${kube_flags[@]}" ${args} "${object}" -o go-template=\""${request}"\")
+    res=$(eval kubectl get "${kube_flags[*]}" ${args} "${object}" -o go-template=\""${request}"\")
```

Comment:
Why are you changing from `[*]` to `[@]` - in my experience `[@]` is almost always more correct

Reply:
I think the change that the check wants is to drop `eval`:

res=$(kubectl get "${kube_flags[@]}" ${args} "${object}" -o go-template=\""${request}"\")

Reply:
Yeah, `eval` inside `$(...)` seems redundant.

Reply:
Thank you for the great feedback! I learnt a few new things about bash scripting when testing your feedback. 

I also removed the escaped quotes `\"` since since we dropped eval.

Reply:
`error: the server doesn't have a resource type "pods selector-test-pod"` 

I think it broke. Will take a look after work.

Reply:
`${args}` is unquoted because we want spaces to matter (it's a hack - bash is bad at arrays).  I am guessing `${object}` is "pods selector-test-pod" which is a hack, but fine.  Unquote that one too.  Leave a comment about why they are unquoted, please :)